docs/sources/explore/_index.md

```diff
@@ -0,0 +1,64 @@
+
+{{< docs-imagebox img="/img/docs/explore/panel_dropdown-7-4.png" class="docs-image--no-shadow" max-width= "650px" caption="Screenshot of the new Explore option in the panel menu" >}}
+
+1. Choose your data source from the dropdown in the top left. [Prometheus](https://grafana.com/oss/prometheus/) has a custom Explore implementation, the other data sources use their standard query editor.
+1. In the query field, write your query to explore your data. There are three buttons beside the query field, a clear button (X), an add query button (+) and the remove query button (-). Just like the normal query editor, you can add and remove multiple queries.
+
+## Split and compare
+
+The split view provides an easy way to compare graphs and tables side-by-side or to look at related data together on one page.
+
+To open the split view:
+
+1. Click the split button to duplicate the current query and split the page into two side-by-side queries.
+   
+It is possible to select another data source for the new query which for example, allows you to compare the same query for two different servers or to compare the staging environment to the production environment.
+
+{{< docs-imagebox img="/img/docs/explore/explore_split-7-4.png" max-width= "950px" caption="Screenshot of the new Explore option in the panel menu" >}}
+
+In split view, timepickers for both panels can be linked (if you change one, the other gets changed as well) by clicking on one of the time-sync buttons attached to the timepickers. Linking of timepickers helps with keeping the start and the end times of the split view queries in sync. It ensures that you’re looking at the same time interval in both split panels.
+
+To close the newly created query, click on the Close Split button.
+
+## Navigate between Explore and a dashboard
+
+To help accelerate workflows that involve regularly switching from Explore to a dashboard and vice-versa, Grafana provides you with the ability to return to the origin dashboard after navigating to Explore from the panel's dropdown.
+
+After you've navigated to Explore, you should notice a "Back" button in the Explore toolbar. Simply click it to return to the origin dashboard. To bring changes you make in Explore back to the dashboard, click the arrow next to the button to reveal a "Return to panel with changes" menu item.
+
+{{< docs-imagebox img="/img/docs/explore/explore_return_dropdown-7-4.png" class="docs-image--no-shadow" max-width= "400px" caption="Screenshot of the expanded explore return dropdown" >}}
+
+## Share shortened link
+
+> **Note:** Available in Grafana 7.3 and later versions.
+
+The Share shortened link capability allows you to create smaller and simpler URLs of the format /goto/:uid instead of using longer URLs containing complex query parameters. You can create a shortened link by clicking on the **Share** option in Explore toolbar. Please note that any shortened links that are never used will be automatically deleted after 7 days.
```

Comment:
```suggestion
The Share shortened link functionality allows you to create smaller and simpler URLs of the format `/goto/:uid` rather than longer URLs that contain complex query parameters. To create a shortened link, click the **Share** option in the Explore toolbar. Any unused shortened link will be automatically deleted after 7 days.
```

Reply:
Took most of the suggestion.

Reply:
Used most of the suggestions.